event_ticket_registration_limit/models/event_ticket.py

```diff
@@ -0,0 +1,21 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+class EventTicket(models.Model):
+    _inherit = 'event.event.ticket'
+
+    max_tickets_per_registration = fields.Integer(
+        string="Max Tickets Per Registration",
+        help="Defines the maximum number of tickets allowed per registration (1-9)."
+    )
```

Comment:
```suggestion
    max_tickets_per_registration = fields.Integer(
        string="Max Tickets Per Registration",
        help="Defines the maximum number of tickets allowed per registration (1-9).",
        default=1
    )
```
I would like to set the default value to 1.